hasher-matcher-actioner/hmalib/common/classification_models.py

```diff
@@ -50,6 +50,11 @@ class BankedContentIDClassificationLabel(ClassificationLabel):
     key: str = field(default="BankedContentIDClassification", init=False)
 
 
+@dataclass(unsafe_hash=True)
+class SubmittedContentClassification(ClassificationLabel):
```

Comment:
namings is hard but we should still try to have consistency here. It looks like all the other `ClassificationLabel` have that in their name not just `Classification`. Also, this is used to check against `additional_fields` yes? can we include the word 'fields` in the name at a minimum?

Reply:
@jeberl, why did we decided not to add `field` here? I was under the impression we were going to but it is possible I am misremembering.